### ISSUE TITLE
Still a packaging problem on 0.3.0

### ISSUE THREAD

**Issue by Toilal** [skatejs/generator-skatejs#9]
```
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module '../../lib/classify'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/toilal/.config/yarn/global/node_modules/generator-skatejs/generators/component/index.js:5:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

It works if i install the generator using the github URL instead of the package name

**Comment by alexlafroscia** [comment]
Heh, whoops! Will fix shortly. Sorry about these issues, I wish there was a better way for me to test what NPM actually uploads to their registry.

**Comment by alexlafroscia** [comment]
Fixed in `0.4.0` (I installed it from NPM myself to make sure).

Thanks for catching that again!
